texttospeech/api/TextToSpeech/TextToSpeech.cs

```diff
@@ -280,5 +295,53 @@ public static void SynthesizeSSMLFile(string ssmlFilePath)
             }
         }
         // [START tts_synthesize_ssml_file]
+
+        // [START tts_synthesize_text_audio_profile]
+        /// <summary>
+        /// Creates an audio file from the text input, applying the specifed
+        /// device profile to the output.
+        /// </summary>
+        /// <param name="text">Text to synthesize into audio</param>
+        /// <param name="outputFile">Name of audio output file</param>
+        /// <param name="effectsProfileId">Audio effect profile to apply</param>
+        /// <remarks>
+        /// Output file saved in project folder.
+        /// </remarks>
+        public static int SynthesizeTextWithAudioProfile(string text,
+                                                         string outputFile,
+                                                          string effectsProfileId) {
+
+            var client = TextToSpeechClient.Create();
+            var response = client.SynthesizeSpeech(new SynthesizeSpeechRequest
+            {
+                Input = new SynthesisInput
+                {
+                    Text = text
+                },
+                // Note: voices can also be specified by name
+                // Names of voices can be retrieved with client.ListVoices().
+                Voice = new VoiceSelectionParams
+                {
+                    LanguageCode = "en-US",
+                    SsmlGender = SsmlVoiceGender.Female
+                },
+                AudioConfig = new AudioConfig
+                {
+                    AudioEncoding = AudioEncoding.Mp3,
+                    // Note: you can pass in multiple effects_profile_id.
```

Comment:
Nit: (optional) – I get that `effects_profile_id` is the field name, but the `snake_case` looks out of place here, right next to both a `PascalCase` and a `camelCase`

```suggestion
                    // Note: you can pass in multiple audio effects profiles.
```